std/experimental/allocator/mallocator.d

```diff
@@ -84,7 +84,7 @@ struct Mallocator
     {
         int* p = null;
         p = cast(int*) A.instance.allocate(int.sizeof);
-        scope(exit) A.instance.deallocate(p[0 .. int.sizeof]);
+        scope(exit) () nothrow @nogc { A.instance.deallocate(p[0 .. int.sizeof]); }();
```

Comment:
This unittest was already marked with `@nogc nothrow`. Why are you wrapping that expression in a lambda?

Reply:
Ops.. sorry about that. Should I do a quick PR for this, or I'll clean it along with a new one?

Reply:
@edi33416 that'd be great thx